app/assets/javascripts/templates/data_portal/modals/modal-download-all.jst.ejs

```diff
@@ -0,0 +1,14 @@
+<div class="c-modal-download-all">
+  <h2 class="title -dark">Download</h2>
+  <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima cumque asperiores nam enim! Veniam reprehenderit possimus, iure labore explicabo ipsum ducimus eligendi</p>
+
+  <form action="#" onSubmit class="c-form js-form">
+    <input type="email" name="email" class="js-email" placeholder="E-mail address" required />
+    <div class="terms-container">
+      <input type="checkbox" name="terms-and-conditions" class="js-terms" /> <span>
```

Comment:
The span should be a label for the checkbox. Currently, screen readers don't say anything about the checkbox.